docs/diagrams/AcademicClassDiagram.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Package Academic {
+Class Academic
+Class Question
```

Comment:
Is there a way to show that Question is an abstract class with UnsolvedQuestion and SolvedQuestion inheriting from it?

Reply:
updated it 
![image](https://user-images.githubusercontent.com/59348559/98434154-1cb4d480-2108-11eb-8b58-a1a6ead256f2.png)